### ISSUE TITLE
[caffe2]ONNX FATAL: [enforce fail at backend.cc:653] . Caffe2 only supports padding 2D Tensor, whereas padding is [0, 3, 3, 0, 0, 3, 3, 0, ]

### ISSUE THREAD

**Issue by laobadao** [pytorch/pytorch#9291]
## Issue description

resnet_v1_50

http://download.tensorflow.org/models/resnet_v1_50_2016_08_28.tar.gz

https://github.com/tensorflow/models/blob/master/research/slim/nets/resnet_v1.py

caffe2 is the latest version, onnx is is the latest version too ,Source code compilation

I convert tensorflow model to onnx and run in caffe2 ,onnx Pad OP convert to Caffe2 PadImage Op ,the attr pads not support 

https://github.com/onnx/tensorflow-onnx

detail : looking this issue :https://github.com/onnx/tensorflow-onnx/issues/66

- PyTorch or Caffe2: Caffe2 is the latest version, onnx is is the latest version too ,
- How you installed PyTorch (conda, pip, source): Source code compilation
